index.js

```diff
@@ -56,21 +72,25 @@ function main(input) {
   }
 }
 
+const evalFn = code => eval(`
+  ${configFileContent}
```

Comment:
This seems a strange to do if in every call. Maybe use globals?

Reply:
I think it offers a better encapsulation to do it in each `eval`, but I may be wrong.